### PR TITLE
Resolve work and dest dirs relative to CWD instead of the root.

### PR DESCRIPTION
**Pull request by garlicnation**

### CODE REVIEW

**Comment by ankon** [comment]
It would be really nice to see this merged ... currently the dest_dir is resolved twice against the root, which really produces funny paths and makes it hard to integrate this tool into a build.

**Comment by justinfagnani** [comment]
Any comments on why this change, or what it fixes?

**Comment by garlicnation** [comment]
`web-component-shards` was tying the destination of the build to the source. This makes it extremely dependent on having a certain sort of project layout. This change brings the flags more in line with expected behavior of a build tool.

File: wcs.js [hunk 1]
```diff
@@ -23,7 +23,7 @@ var WebComponentShards = function WebComponentShards(options){
   this.bowerdir = options.bowerdir;
   this.shared_import = options.shared_import;
   this.sharing_threshold = options.sharing_threshold;
-  this.dest_dir = url.resolve(this.root, options.dest_dir);
+  this.dest_dir = path.resolve(options.dest_dir) + "/";
```
Comment (justinfagnani) [inline wcs.js]: use path.join rather than string concatenation?

**Comment by justinfagnani** [comment]
LGTM

**Pull request merged**